modules/oxide/src/less/theme/components/toolbar-button/toolbar-label.less

```diff
@@ -0,0 +1,16 @@
+//
+// Toolbar label
+// When making changes here, please review menubar/menubar.less and menubar/menubar-button.less
```

Comment:
Why would one need to review menubar? Maybe remove this line?

Reply:
done